### DISCUSSION TITLE
Schneider electric smoke detector WISER CCT599001

### DISCUSSION THREAD

**Discussion by ulferi** [Koenkk/zigbee2mqtt#14738] [Devices]
HI!
Anyone working on Schneider electric WISER CCT599001 smoke detector
Detected as Scneider electric W599001

Is it possible to get it to work?
Can you share your settings for that one?
//Ulf

**Comment by ulferi** [comment]
Can someone please help me to do this integration!

**Reply by senna1992** [reply]
https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

if you follow this instruction,
you can do it!


remember to restart home assistant after changing your new converter =)

**Comment by senna1992** [comment]
Update: anyone had issues with false alerts on the detector while running on zigbee2mqtt?


I moved mine (on Thursday) over from wiser hub and now, be it a coincidence or not - today I had two alerts 4 hours apart, duration over 2min each. without any signs or smell of fire or heat inside the basement.


Great to know, that they interconnect to their brothers directly when seeing each other on zigbee. So I am now officially awake. At 11.30pm. 😅

**Comment by huco95** [comment]
I have one of this devices an I'm planning to add it with zigbee2mqtt. Did you make it work? @senna1992 @ulferi

**Comment by senna1992** [comment]
Does it work?
Yes.


Does it actuate false alarms?
maybe.

Does the internally managed chaining work when using z2m? Definitely.

Did It wake up my neighbors?
Very well, Twice


No such issues while pairing to wiser hub v2 and using the hacs integration for it

________________________________

**Reply by djGLiTCH** [reply]
Would you mind sharing how you got your wiser hub v2 to connect to the HACS integration?

The standard IP address used to get the secret key isn't working for me (192.168.8.1/secret), and I've tried everything I can think of with no success.

I already have the HACS integration ready, just can't get the secret key to complete the connection and bring my smoke alarms online.

Wiser Hub model is CLP501902
